pkg/util/git.go

```diff
@@ -0,0 +1,38 @@
+package util
+
+import (
+	"fmt"
+
+	git "gopkg.in/src-d/go-git.v4"
+	gitconfig "gopkg.in/src-d/go-git.v4/config"
+)
+
+// GetRepositoryRemotes returns a map containing the remote names and the URLs they
+// point to.
+func GetRepositoryRemotes(repo *git.Repository) (map[string][]string, error) {
```

Comment:
This may be dumb question but isn't it "remoteName -> remoteUrl" and not "remoteName -> []remoteUrl" ?

Reply:
That's a good question @vijtrip2 ! Me too before writing this patch, I thought that a remote can only have one URL, but while using go-git I found out that a remote can have multiple URLs (multiple URLs for push and multiple URLs for pull). This gist shows how to set multiple URLs for a remote using `git` https://gist.github.com/bjmiller121/f93cd974ff709d2b968f

Reply:
That is super cool!